common/src/command.rs

```diff
@@ -17,13 +95,21 @@ impl<T: CommandType> CommandTypeDynamic for T {
 
 #[derive(Debug, Clone)]
 pub struct Speak(pub String);
+
+#[derive(Debug, Clone)]
+pub struct SpeechPriority(pub Priority);
```

Comment:
this shouldn't be a command at all, much less be in the handlers, that's far too verbose. A default priority is used, and the user can optionally change that in the configuration file, what gets what priority.

Reply:
You are right that this should not be its own command. I think the priority should be set in the speech command.

This should *not* be set by config file; this changes based on whether the event is a notification, focus event, progress bar update, or text coming from a foreign source (this `odiliactl`-style pushing).

Reply:
yep, that's right, but in no way should this be a command or something like that, so I'll close this convo as resolved